src/vs/workbench/browser/parts/editor/breadcrumbsControl.ts

```diff
@@ -97,6 +98,9 @@ class OutlineItem extends BreadcrumbsItem {
 		this._disposables.add(toDisposable(() => { renderer.disposeTemplate(template); }));
 	}
 
+	supportDrag(): boolean {
+		return false;
```

Comment:
I believe outline items should also support DND. They can use the file-uri and the postion anchor, like file:///some/path.txt#L12:34 for line 12, column 34 of file some/path.txt